Code/Editor/Platform/Windows/Editor/Core/QtEditorApplication_windows.cpp

```diff
@@ -118,10 +118,24 @@ namespace Editor
                 }
                 return true;
             }
+#if defined(CARBONATED)
+            else if (msg->message == WM_KEYDOWN || msg->message == WM_KEYUP)
+            {
+                return false; // pass through unhandled so that WM_CHAR is generated
+            }
+            else if (msg->message == WM_CHAR)
+            {
+                // Process raw WM_CHAR event sending keycode to a LyShine UiTextInputComponent in Editor game mode
+                AzFramework::RawInputNotificationBusWindows::Broadcast(
+                    &AzFramework::RawInputNotificationsWindows::OnRawInputCodeUnitUTF16Event, msg->wParam);
+                return true;
```

Comment:
This now ignores the "filterInputEvents" event entirely.  Is this intended?   Does the Editor respond to ctrl+ etc shortcut keys and all that, in addition to accepting keyboard input during gameplay and in the text input box?

Reply:
> This now ignores the "filterInputEvents" event entirely. Is this intended?

Yes, it is intended. The commot introducing filterInputEvents was wrong for character input events.

> Does the Editor respond to ctrl+ etc shortcut keys and all that, in addition to accepting keyboard input during gameplay and in the text input box?

Ctrl+etc. shortcut events are blocked in Editor game mode, except for legal "Ctrl+a", "Ctrl+x", "Ctrl+c", "Ctrl+v", etc while in text input element.

Reply:
What about when the editor is _not_ in game mode?  Do the shortcuts etc still function?

Reply:
No changes to Editor mode here,.
Please note that my chages are inside the block
```
if (GetIEditor()->IsInGameMode())
{
}
```
while otherwise code of this event handler always
` return false;`
which is right.

Reply:
ah I see - thank you